skins/laika/tests/unit/components/pages/donation_form/AddressType.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import AddressType from '@/components/pages/donation_form/AddressType.vue';
+import Buefy from 'buefy';
+
+const localVue = createLocalVue();
+localVue.use( Buefy );
+
+describe( 'AddressType.vue', () => {
+
+	it( 'emits field changed event on blur', () => {
+		const wrapper = mount( AddressType, {
+				localVue,
+				mocks: {
+					$t: () => { },
+				},
+			} ),
+			event = 'address-type',
+			company = wrapper.find( '#company' );
+		company.trigger( 'change' );
+		const person = wrapper.find( '#personal' );
+		person.trigger( 'change' );
+		const anon = wrapper.find( '#anonymous' );
+		anon.trigger( 'change' );
+		expect( wrapper.emitted( event ) ).toHaveLength( 3 );
```

Comment:
You _could_ check if the events submitted the right type data, but having at least the basic event handling checked is better than nothing.

Reply:
Good that you mentioned that, because when I added those assertions I realized triggering a change event doesn't work for radio buttons. You have to trigger a click event, even though the component is triggering a change 🤷‍♀